jina/resources/helloworld.flow.query.yml

```diff
@@ -6,11 +6,9 @@ with:
 pods:
   encode:
     uses: $RESOURCE_DIR/helloworld.encoder.yml
-    parallel: $PARALLEL
   index:
     uses: $RESOURCE_DIR/helloworld.indexer.yml
     shards: $SHARDS
     separated_workspace: true
     polling: all
-    uses_after: $RESOURCE_DIR/helloworld.reduce.yml
```

Comment:
I guess that was an artifact from old!?

Reply:
reduce is needed if used in parallel right? but it may use `_reduce` anyhow? why was it needed to be removed?

Reply:
It just does not provide any value and was not working before. So I thought removing complexity from a `hello-world` is a good thing.

Reply:
All in for reducing complexity, and since u found the problem with parallelism in encoder, it makes it more obvious, reducer only comes to play when parallelism is up

Reply:
this is required, do not remove